packages/big-design/src/components/Form/spec.tsx

```diff
@@ -17,12 +17,14 @@ test('forwards ref', () => {
   expect(form).toBe(ref.current);
 });
 
-test('calls onSubmit', () => {
+test('calls onSubmit', async () => {
   const onSubmit = jest.fn();
   const { container } = render(<Form onSubmit={onSubmit} />);
-  const form = container.querySelector('form') as HTMLFormElement;
+  const form = container.querySelector('form');
```

Comment:
If you give the `Form` a `name` attribute you should be able to use `findByRole('form')` query.